src/System.Private.TypeLoader/src/Internal/Runtime/TypeLoader/DispatchCellInfo.cs

```diff
@@ -22,6 +22,7 @@ internal enum DispatchCellType
         VTableOffset = 0x2,
     }
 
+#pragma warning disable 0649 // Disable warning about unused field. Some fields in struct below are only manipulated from native code
```

Comment:
I usually just put StructLayout.Sequential on these. It shuts up the compiler and I don't need a comment to explain myself. It also prevents me from forgetting #pragma warning enable.